docs/api/tracker.rst

```diff
@@ -7,6 +7,13 @@
 Tracker
 =======
 
+.. edit-link::
+
+Trackers maintain the state of the a dialogue and can be featurized for machine
+learning algorithms right out of the box. The dialogue between the assistant and the
```

Comment:
Don't know if we need the marketing/SEO sentence starting off the section 😄I think `Trackers maintain the state of the dialogue` is fine

Reply:
👍